geometry/shapes.py

```diff
@@ -432,3 +432,45 @@ def intersects(obj1, obj2):
             return False
 
     return True
+
+
+class Triangle(Polyhedron):
+    def __init__(self, *args):
+        super(Triangle, self).__init__()
+
+        self._vertices = []
+        input_shape = np.array(args[0]).shape
+
+        if len(args) != 3:
```

Comment:
If I pass in a 3x3 array, you won't catch an error. You need to explicitly check that it's a 3x1 or 1x3.